moses/moses/main/table-problems.h

```diff
@@ -69,6 +70,8 @@ class table_problem_base : public problem_base
     type_tree table_type_signature;
     type_tree cand_type_signature;
     type_node output_type;
+
+	AtomSpace* _as;
```

Comment:
The indentation is inconsistent with the file style. You may either fix it now to be space, or in a subsequent commit fix the remaining file to be (smart/mixed) tabs. (I prefer the latter but it's up to you).

Reply:
Oh, sorry I meant "subsequent PR" :-\

No big deal for this time of course it can go in that PR.